src/core/network/createNetwork.js

```diff
@@ -106,8 +106,20 @@ export default (config, logger, lifecycle, networkStrategy) => {
           return handleResponsePromise;
         })
         .catch(error => {
-          lifecycle.onResponseError(error);
```

Comment:
This is why errors from `onResponseError` were not being suppressed. Nothing was handling errors from this promise.